.requirements

```diff
@@ -12,7 +12,7 @@ LUA_RESTY_WEBSOCKET=60eafc3d7153bceb16e6327074e0afc3d94b1316 # 0.4.0
 ATC_ROUTER=b0d5e7e2a2ca59bb051959385d3e42d96c93bb98 # 1.2.0
 
 KONG_MANAGER=nightly
-NGX_WASM_MODULE_BRANCH=main
+NGX_WASM_MODULE=nightly
 WASMER_VERSION=3.1.1
 WASMTIME_VERSION=8.0.1
 V8_VERSION=10.5.18
```

Comment:
not originally from this PR, but let's use the new naming convention in requirements for other components as well (rstrip _BRANCH, _VERSION etc)
```suggestion
WASMER=3.1.1
WASMTIME=8.0.1
V8=10.5.18
```